collector/client.go

```diff
@@ -10,14 +10,15 @@ import (
 	"net/http"
 	"strconv"
 	"strings"
-
+	b64 "encoding/base64"
 	"github.com/boynux/squid-exporter/types"
 )
 
 /*CacheObjectClient holds information about squid manager */
 type CacheObjectClient struct {
 	hostname string
 	port     int
+	encryptedLoginPass string
```

Comment:
Please change the variable name to something like `basicAuthString` or something like that.